buildSrc/src/main/groovy/com/uber/okbuck/core/dependency/VersionlessDependency.groovy

```diff
@@ -9,14 +9,21 @@ class VersionlessDependency {
 
     final String group
     final String name
+    final String classifier
 
-    VersionlessDependency(ModuleVersionIdentifier identifier) {
+    VersionlessDependency(ModuleVersionIdentifier identifier, String artifactType, String artifactClassifier) {
         group = identifier.group
         name = identifier.name
+        if (artifactType == "jar") {
```

Comment:
This is q very custom hack that only works for jar artifacts. I do not like how this is currently setup. aars can have classifiers as well when they are external dependencies.

I think the proper fix is to ignore the artifactType and classifier if it is not an external dependency.